File: examples/lh-server/python/10-interrupts/tasks.py
from typing import Any


async def start_collection() -> None:
    print("starting-collection-of-underpants")


async def ship_item(underpant_owner: str) -> str:
    result = f"Successfully collected underpant from {underpant_owner}"
    print(result)
    return result


async def profit(underpants: list[Any]) -> str:
    result = f"Collected {len(underpants)} underpants!"
    print(result)
    return result

File: examples/lh-server/go/10-interrupts/tasks.go
package main

import "fmt"

func startCollection() string {
	fmt.Println("Starting collection of underpants")
	return ""
}

func shipItem(underPantOwner string) string {
	result := fmt.Sprintln("Successfully collected underpants from ", underPantOwner)
	fmt.Println(result)
	return result
}

func profit(underpants *[]string) string {
	result := fmt.Sprintf("Collect %d underpants", len(*underpants))
	fmt.Println(result)
	return result
}
